packages/react/src/components/MeterChartCard/MeterChartCard.story.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { text, number, select, boolean, object } from '@storybook/addon-knobs';
+import { layout05 } from '@carbon/layout';
+
+import { CARD_SIZES } from '../../constants/LayoutConstants';
+import { getCardMinSize } from '../../utils/componentUtilityFunctions';
+
+import MeterChartCardREADME from './MeterChartCard.mdx';
+import MeterChartCard from './MeterChartCard';
+
+export default {
+  title: '1 - Watson IoT/MeterChartCard',
```

Comment:
These stories for the cards should be nested inside the Card story. 
```suggestion
  title: '1 - Watson IoT/Card/MeterChartCard',
```